### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pranavraja99**
switched to AgML 0.2.3 for data loading and provided instructions to use with custom data, users can now also easily normalize (center and scale) their custom data

### CODE REVIEW

File: example.ipynb [hunk 1]
```diff
@@ -92,9 +92,54 @@
...
+      "can use `agml.data.source(<name>).citation_summary().`\n",
+      "\n",
+      "You can find your dataset at /data/pvraja/DeformableCNN-PlantTraits/autonomous_greenhouse_regression.\n",
+      "================================================================================\n"
+     ]
+    }
+   ],
    "source": [
     "import agml\n",
     "loader = agml.data.AgMLDataLoader('autonomous_greenhouse_regression', dataset_path = './')"
```
Comment (amogh7joshi) [inline example.ipynb:145]: Since you're just downloading the dataset and then manually using it, use the following (no need to instantiate a loader):

```
agml.data.download_public_dataset('autonomous_greenhouse_regression', location = '.')
```

File: example.ipynb [hunk 2]
```diff
@@ -237,15 +290,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Instantiate the PyTorch datalaoder the autonomous greenhouse dataset.\n",
```
Comment (amogh7joshi) [inline example.ipynb:297]: Typos 😅

File: example.ipynb [hunk 3]
```diff
@@ -413,15 +672,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Instantiate the PyTorch datalaoder the autonomous greenhouse dataset.\n",
```
Comment (amogh7joshi) [inline example.ipynb:681]: Typo
